internal/uvm/create.go

```diff
@@ -5,17 +5,30 @@ import (
 	"os"
 	"path/filepath"
 	"runtime"
+	"strings"
 
 	"github.com/Microsoft/hcsshim/internal/guid"
 	"github.com/Microsoft/hcsshim/internal/hcs"
 	"github.com/Microsoft/hcsshim/internal/mergemaps"
 	"github.com/Microsoft/hcsshim/internal/schema2"
 	"github.com/Microsoft/hcsshim/internal/schemaversion"
 	"github.com/Microsoft/hcsshim/internal/uvmfolder"
+	"github.com/Microsoft/hcsshim/internal/wclayer"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/sirupsen/logrus"
 )
 
+type PreferredRootFSType int
+
+const (
+	PreferredRootFSTypeDefault = 0
```

Comment:
I don't like code like this. Default means you have to have some IF that transforms the code into initrd or vhd based on some case (by default). Why not just start InitRd = 0 so it is the default and if you want VHD you have to specify it

Reply:
That's updated now.